ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1062,6 +1062,38 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
             break;
 #endif
 
+#if AC_RALLY == ENABLED
+        case MAV_CMD_NAV_RALLY_POINT: //not including home
```

Comment:
MAVLink documentation doesn't mention what this should be used for, but following the history it looks like it was added for the new rally protocol and not exactly to go to a rally point. I think this needs to be clarified in MAVLink first.

Reply:
Same nothing define.
How long should we wait for Mavlink then...?

Reply:
Is there any discussion in MAVLink about this?

Reply:
Nothing more than this https://github.com/mavlink/mavlink/pull/683 from March 2017.

Reply:
As written in https://github.com/mavlink/mavlink/pull/683#discussion_r107409146, these new commands were intended to be used in MISSION_ITEM and not in COMMAND_LONG.

Reply:
Changing the MAVLink message needs to be discussed in the MAVLink repo. We can add a new MAVLink message, but I don't think that's needed and will only bring more confusion.

Reply:
Is there some question here someone if looking for me to answer?

Reply:
Maybe this is not the place to make this question but I can't find the answer in anyplace. I can't connect my Arducopter 2.6 to Qgroundcontrol (mission planner is ok) Any idea? Thanks, I'm new in this world.